README.md

```diff
@@ -229,6 +229,34 @@ end
 
 Note that InSpec AWS assumes full compatibility with the underlying AWS SDK and unsupported operations will cause failures.  Therefore, depending on the external provider implementation your mileage may vary!
 
+### `aws_retry_limit` and `aws_retry_backoff`
+
+In certain cases AWS may implement rate limiting. In order to mitigate this issue the `Retry Limit` and `Retry Backoff` can be set in two ways:
+
+#### 1) Environment Variables
+Setting `AWS_RETRY_LIMIT` and `AWS_RETRY_BACKOFF` environment variables will be implemented at session level
+
+```bash
+export AWS_RETRY_LIMIT=5
+export AWS_RETRY_BACKOFF=5
+```
+
+#### 2) Inspec Control
+Inspec AWS resources now support setting the Retry Limit and Retry Backoff at control level as shown below.
+
+```
+  describe aws_config_recorder(recorder_name: aws_config_recorder_name, aws_retry_limit=5, aws_retry_backoff=5) do
+    it { should exist }
+    its('recorder_name') { should eq aws_config_recorder_name }
+  end
+```
+
+ #####The `aws_retry_limit` and `aws_retry_backoff` precedence:
+   1. Set at Inspec control level.
+   2. Set at `.envrc`/ Environment variable level.
```

Comment:
`.envrc` is optional, can we just mention setting these as case-insensitive environment variables here e.g. 
```
export AWS_RETRY_LMIIT=...
export aws_retry_limit=...
```